metafix/src/main/java/org/metafacture/metafix/FixMethod.java

```diff
@@ -655,8 +657,15 @@ public void apply(final Metafix metafix, final Record record, final List<String>
         public void apply(final Metafix metafix, final Record record, final List<String> params, final Map<String, String> options) {
             record.transform(params.get(0), s -> s.toUpperCase());
         }
+    },
+    uri_encode {
+        @Override
+        public void apply(final Metafix metafix, final Record record, final List<String> params, final Map<String, String> options) {
+            record.transform(params.get(0), PERCENT_ESCAPER::escape);
+        }
     };
 
+    private static final PercentEscaper PERCENT_ESCAPER = new PercentEscaper("", false);
```

Comment:
From what I understood, the conclusion from #273 was to basically use the [metamorph implementation](https://github.com/metafacture/metafacture-core/blob/master/metamorph/src/main/java/org/metafacture/metamorph/functions/URLEncode.java), which uses `java.net.URLEncoder`. If there is no specific reason for using `PercentEscaper` here, I think using the standard Java class would be preferable.

Reply:
In lobid-resources I added a [UrlEscaper}(https://github.com/hbz/lobid-resources/blob/master/src/main/java/org/lobid/resources/UrlEscaper.java) which also complies to RFC  3986, using an older `com.google.gdata.util.common.base.PercentEscaper`. It hasn't make it to metafacture (yet).